server/src/main/java/org/opensearch/gateway/remote/RemoteClusterStateService.java

```diff
@@ -596,6 +602,11 @@ private void writeMetadataManifest(String clusterName, String clusterUUID, Clust
             blobStoreRepository.getCompressor(),
             FORMAT_PARAMS
         );
+        logger.debug(
+            "Metadata manifest file [{}] written during [{}] phase. ",
```

Comment:
Maybe we can add details about if global metadata updated, no of indices updated as well?

Reply:
I have added log line which will tell us how many index got updated/skipped and if global metadata has been changed or not. 

`logger.info(
                "writing cluster state for version [{}] took [{}ms]; "
                    + "wrote metadata for [{}] indices and skipped [{}] unchanged indices, global metadata updated : [{}]",...`